src/example_ddp_anndata.py

```diff
@@ -0,0 +1,151 @@
+import random
```

Comment:
Is this useful anymore if we are no longer considering an iterable-style dataset?  If Yes -> move these individual classes into there right place in the package hierarchy, if not… vaporize